lib/assert.js

```diff
@@ -109,7 +109,7 @@ assert.ok = ok;
 // The equality assertion tests shallow, coercive equality with
 // ==.
 // assert.equal(actual, expected, message_opt);
-
+/* eslint-disable no-restricted-properties */
```

Comment:
This line was already in use for `deepEqual` and `notDeepEqual` below, it's now being used for 4 functions (it's re-enabled below in [L136](https://github.com/nodejs/node/pull/10698/commits/fb9c76fdcebb4c5319e3595aa1764a91109561bf#diff-f76cd6d4ba3dad8b6b9a393af1fce15bL136)). I think we'd need 8 `eslint-disable-next-line` lines if we did it individually.